apps/admin/src/components/events/options/apply/index.tsx

```diff
@@ -64,6 +67,11 @@ const ApplyOption = () => {
       onSuccess: (data) => {
         queryClient.invalidateQueries({ queryKey: ['AppliedTicket', eventId] });
       },
+      onError: (error: any) => {
+        const comment = error.response.data.reason;
+        setToast({ comment: comment });
```

Comment:
```suggestion
        setToast({ comment: comment, type:'error' });
```
뭐 이런거 빨간색으로 뜨는거 있을거야

Reply:
헐 알았으 지금 짜잘한거 고치고있는데 그거랑같이올릴겡